systemtest/src/test/java/io/strimzi/systemtest/operators/user/UserST.java

```diff
@@ -158,7 +159,23 @@ void testBigAmountOfScramShaUsers(ExtensionContext extensionContext) {
         String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
         String userName = mapWithTestUsers.get(extensionContext.getDisplayName());
 
-        createBigAmountOfUsers(extensionContext, userName, "SCRAM_SHA");
+        createBigAmountOfUsers(extensionContext, userName, "SCRAM_SHA", 100);
+    }
+
+    @Tag(SCALABILITY)
+    @ParallelTest
```

Comment:
Should these kind of tests really run in parallel?

Reply:
I am not creator of them, just this one, but why not? 
It stresses UO a bit more. IIUC single kafka cluster is being used for all of these tests.

But maybe @see-quick or @im-konge may share/see your concern?

Reply:
I think the idea of running load tests in parallel on a shared environment is. bit weird ... btu lets see how it works 🤷‍♂️